corehq/apps/hqadmin/tasks.py

```diff
@@ -154,3 +157,8 @@ def _mass_email_attachment(name, rows):
         'file_obj': csv_file,
     }
     return attachment
+
+
+@periodic_task_when_true(settings.IS_SAAS_ENVIRONMENT, run_every=crontab(minute="0", hour="*/4"), queue='background_queue')
```

Comment:
Could this ever take more than 4 hours to run? Might be safer to have this run only once a day, during off period

Reply:
no...this takes a few minutes to run...max 30